test/central-config-enabled.js

```diff
@@ -36,7 +26,8 @@ test('remote config enabled', function (t) {
   })
 
   server.listen(function () {
-    agent = require('..').start({
+    // agent = require('..').start({
```

Comment:
Drop this commented line?

Reply:
we regret the error -- we also regret the next occurrence of this error -- I can be comment blind towards the end of a feature 🤦